doc/source/groupby.rst

```diff
@@ -955,6 +959,42 @@ will be (silently) dropped. Thus, this does not pose any problems:
 
    df.groupby('A').std()
 
+.. note::
```

Comment:
this is for another issue?

Reply:
PR #18953 ?

Reply:
Yes, I should have made this a separate branch on my fork and separate pull request.

I will make the updates per your notes above.

Reply:
I created a clean pull request for this fix: #19215